pkgs/development/python-modules/dagger-io/default.nix

```diff
@@ -0,0 +1,60 @@
+# pkgs = import (fetchTarball {
+# 	url = "https://github.com/NixOS/nixpkgs/archive/nixpkgs-unstable.tar.gz";
+# 	sha256 = "sha256:0s4hvb6zgbi3nqnh2r4q129q4672fv65cy02mxchddz7p5hg5sma";
+# }) {};
+
+{ buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, poetry-core
+, anyio
+, attrs
+, cattrs
+, graphql-core
+, gql
+, httpx
+, beartype
+, platformdirs
+, typing-extensions
+, rich
+, typer
+, strawberry-graphql
+, shellingham
+, colorama
```

Comment:
this should be alphabetically ordered.  Except for the lib attribute that should be at the top.